src/components/Auth/Login/Login.react.js

```diff
@@ -159,6 +160,26 @@ class Login extends Component {
     }
   };
 
+  callGetAdmin = accessToken => {
+    this.props.actions
+      .getAdmin(accessToken)
+      .then(({ response }) => {
+        let { showAdmin, accepted } = response;
+        if (accepted) {
+          cookies.set('isAdmin', showAdmin, {
+            path: '/',
+            domain: cookieDomain,
+          });
+        }
+        this.setState({
```

Comment:
There is no `isAdmin` state defined. Here, call an action to set `isAdmin` to true in the Redux state.

Reply:
Oh, missed that. Made the changes